apps/site/assets/ts/hooks/usePredictionsChannel.ts

```diff
@@ -55,6 +55,12 @@ export const parsePrediction = (
     "status",
     "vehicle_id"
   ]),
+  arrival_time: prediction.arrival_time
+    ? new Date(prediction.arrival_time)
+    : null,
+  departure_time: prediction.departure_time
+    ? new Date(prediction.departure_time)
+    : null,
```

Comment:
Can you update the tests to ensure we are converting arrival and departure times correctly?